<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Set_time_condition</class>
 <widget class="QDialog" name="Set_time_condition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1021</width>
    <height>533</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1021</width>
    <height>533</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pBtnUseOnOff1_set_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>501</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set_irrigation_time_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>251</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>개별시간</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDispDate_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1001</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>작동조건 선택</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set_irrigation_time_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>241</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>일정시간</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set_irrigation_time_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>광량</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set_irrigation_time_6">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>80</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>지습</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set_irrigation_time_7">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>330</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>35</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_set_irrigation_time_8">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0 J/cm2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mon">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>61</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(85, 255, 255)</string>
   </property>
   <property name="text">
    <string>월</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tue">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>420</y>
     <width>61</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(85, 255, 255)</string>
   </property>
   <property name="text">
    <string>화</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_wen">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>420</y>
     <width>61</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(85, 255, 255)</string>
   </property>
   <property name="text">
    <string>수</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_thu">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>420</y>
     <width>61</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(85, 255, 255)</string>
   </property>
   <property name="text">
    <string>목</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fri">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>420</y>
     <width>61</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(85, 255, 255)</string>
   </property>
   <property name="text">
    <string>금</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sat">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>420</y>
     <width>61</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(85, 255, 255)</string>
   </property>
   <property name="text">
    <string>토</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sun">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>420</y>
     <width>61</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(85, 255, 255)</string>
   </property>
   <property name="text">
    <string>일</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pBtnUseOnOff1_set_15</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>set_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_irrigation_time_3</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>set_time_con_indvd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_irrigation_time_4</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>set_time_con_certain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_irrigation_time_5</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>set_time_con_light()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_irrigation_time_6</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>set_time_con_humid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>816</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_irrigation_time_8</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>set_time_con_j()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_set_irrigation_time_7</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>set_time_con_h_per()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>784</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_mon</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>click_day_of_week()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_tue</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>click_day_of_week()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_wen</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>click_day_of_week()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_thu</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>click_day_of_week()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>755</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_fri</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>click_day_of_week()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>823</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sat</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>click_day_of_week()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>883</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>888</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sun</sender>
   <signal>clicked()</signal>
   <receiver>Set_time_condition</receiver>
   <slot>click_day_of_week()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>956</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>971</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>set_cancel()</slot>
  <slot>set_time_con_save()</slot>
  <slot>set_time_con_indvd()</slot>
  <slot>set_time_con_certain()</slot>
  <slot>set_time_con_light()</slot>
  <slot>set_time_con_humid()</slot>
  <slot>set_time_con_j()</slot>
  <slot>set_time_con_h_per()</slot>
  <slot>click_day_of_week()</slot>
 </slots>
</ui>
